### DISCUSSION TITLE
Make Molten-nvim work with conda

### DISCUSSION THREAD

**Discussion by zorjo** [benlubas/molten-nvim#177] [Q&A] [answered]
Make this work with the kmontocam/nvim-conda plugin. 
and hotswap conda environmets while evaluating. 
Environment : Windows 10

**Comment by benlubas** [comment]
> hotswap conda environmets while evaluating

afaik this is not possible

**Reply by zorjo** [reply]
Well, even launching nvim from a conda activated shell does not work. Is it possible to launch a jupyter kernel from conda and connect to that from Molten?
In general, how would you reccomend to connect with conda?

**Reply by benlubas** [reply, marked as answer]
Same instructions from here would probably work with conda: https://github.com/benlubas/molten-nvim/blob/main/docs/Virtual-Environments.md

If you wanted to start a kernel from the command line you can connect to it following the instructions here: https://github.com/benlubas/molten-nvim/blob/main/docs/Advanced-Functionality.md#connecting-to-external-kernels

**Reply by zorjo** [reply]
I'll confirm if this works and close the discussion if that's the case

**Reply by zorjo** [reply]
Both Methods have been tested and seem to be working. There's some lag, and the editor froze, but I'm not sure it's the fault of this plugin. Just add conda on the Virtual environments page for future reference. Thanks!

**Reply by benlubas** [reply]
If the editor froze it's probably this issue: #27 

If you want to add your reproduction to the thread that would be helpful, I haven't had a ton of time to look into the issue unfortunately.

Installing the kernel with `--user` usually fixes it.
